js/shell.js

```diff
@@ -8,7 +8,8 @@ class Shell {
 
     localStorage.directory = 'root'
     localStorage.history = JSON.stringify('')
-
+    localStorage.historyIndex = -1 // Solves undefined command on refreshing the website
+    localStorage.goingThroughHistory = 'false' // To prevent down arrow traversal when not required
```

Comment:
this should be a boolean and update/checked accordingly

Reply:
done